### PR TITLE
WSTEAM1-128: Byline Location

### PR DESCRIPTION
**Pull request by mfonofm**
Resolves #WSTEAM1-128

**Overall change:**
Adds functionality to optionally include location in the byline

**Code changes:**

- Add `location` to byline
- Add styles for `location` and `reportingFrom`
- Add and update unit tests
- Add `bylineWithLinkAndLocation` fixture
- Add `LinkAndLocationByline` to Storybook

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project
- [x] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/pages/ArticlePage/Byline/index.tsx [hunk 1]
```diff
@@ -161,6 +179,31 @@ const Byline = ({ blocks, children }: PropsWithChildren<Props>) => {
...
+              <span
+                css={[
+                  BylineCss.reportingFromText,
+                  getSansRegularItalic(service),
+                  getBrevier(script),
+                ]}
+              >
+                {reportingFromTranslated}
+              </span>
+              <br />
```
Comment (greenc05) [inline src/app/pages/ArticlePage/Byline/index.tsx]: Can we do this with CSS please?

File: src/app/pages/ArticlePage/Byline/index.tsx [hunk 2]
```diff
@@ -161,6 +179,31 @@ const Byline = ({ blocks, children }: PropsWithChildren<Props>) => {
             </a>
           </li>
         ) : null}
+        {location ? (
+          <li>
+            <p css={[BylineCss.location]}>
```
Comment (greenc05) [inline src/app/pages/ArticlePage/Byline/index.tsx]: Not sure we need a paragraph element?

Comment (mfonofm) [reply src/app/pages/ArticlePage/Byline/index.tsx]: Yep, I've changed it to a span

File: src/app/pages/ArticlePage/Byline/index.tsx (hunk 1 above)
Comment (mfonofm) [reply src/app/pages/ArticlePage/Byline/index.tsx]: Yep, I've removed the br tag and applied block styling to the `location`

File: src/app/pages/ArticlePage/Byline/index.tsx [hunk 3]
```diff
@@ -161,6 +179,30 @@ const Byline = ({ blocks, children }: PropsWithChildren<Props>) => {
             </a>
           </li>
         ) : null}
+        {location ? (
+          <li>
+            <span css={[BylineCss.location]}>
```
Comment (amoore108) [inline src/app/pages/ArticlePage/Byline/index.tsx]: Super minor, but this can be taken out of an array if its just the 1 CSS object.

**Review by amoore108 — APPROVED** [review]
Nice one! 👍🏼

File: src/app/pages/ArticlePage/Byline/index.tsx (hunk 3 above)
Comment (mfonofm) [reply src/app/pages/ArticlePage/Byline/index.tsx]: Good spot

**Review by greenc05 — APPROVED** [review]
Thanks for the changes Mfon

**Pull request merged**